### PR TITLE
February 9 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | ef231a9 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-401.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by robertengaxway — APPROVED** [review]

File: content/en/docs/amplify_relnotes/20240209_amplify.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+#### Feature updates for Product Foundry
+
+* **Selected Version UX Improvement** - A button has been added to display the currently selected Prodcut version if the selected version should scroll out of view.  up-arrow button which is displayed if the user scrolls away from the top of the list.
+* **Base product plan price one time fee or recurring fee** - The paid plan has been enriched to allow provider if he want his consumer to pay a one-time fee or a recurring fee for the base price plan. Refer to [Configure paid plan - base price](/docs/manage_product_foundry/manage_product_plans#configure-a-quota)
+
+#### Bug fixes for Product Foundry
+
+* **Long Marketplace name overlap the status value when publishing product** - A long Marketplace name is now truncated before it overlap the publication status.
+* **Line break issue in description fields** - Editing a product description allow to enter line break. Those breaks are correctly render after saving the product.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240209_amplify.md:120]: **Line break issue in description fields** - The product description can now handle line breaks.

**Comment by lbadenhop** [comment]
Spell checked. Will review/edit once @georgiana-caras  approves content.

File: content/en/docs/amplify_relnotes/20240209_amplify.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+* **Resource version not visible when creating a plan quota** - When creating a plan quota, the resource name is enriched to display: *{assetTitle}/{resourceTitle} - {stageTitle} - {stateName} - {Version}*. When viewing the plan quota details, these information are displayed in a table format.
+* **Log audit concern** - When Axway is upgrading the model and that update triggers event that are displayed in the event logs, those events are now identified as belonging to **Axway Internal Admin** user.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240209_amplify.md:122]: **System initiated events logged with a platform user account** - All actions initiated by the system such as upgrade  / migration codes are now now reported in the Activity events as being performed by "Axway Internal Admin" user.

File: content/en/docs/amplify_relnotes/20240209_amplify.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+* **Cannot delete a stage** - After removing a stage to an environment, it was not possible to remove the stage because of some remaining reference in the environment. An upgrade procedure has been run to clean up all the bad references. This upgrade triggers some Stage update event in the activity log associated to the Axway Internal Admin user.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240209_amplify.md]: **Impossible to delete a stage** - We have fixed an issue where it was not possible to remove a stage, even after it was no longer in use. This was caused by some remaining references on the environments that were still pointing to the stage. An upgrade procedure has been run to clean up all the invalid references. This upgrade will trigger a stage update event in the Activity log that is now reported as performed by the Axway Internal Admin user.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Review by cbordier — APPROVED** [review]
Looks good

**Pull request merged**